### PR TITLE
Add OPEV-0015 Document

### PR DESCRIPTION
**Pull request by puerco**
This PR checks in the document describing the changes required to implement [OPEV-0015: Expansion of the Vulnerability Field](https://github.com/openvex/community/issues/15)

### CODE REVIEW

File: enhancements/opev-0015.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  "@id": "https://openvex.dev/docs/public/vex-6e8cc8c4d0c38e9e9a60949e5dc0279684d8ad5f6711d9a12bb52247b6cc7271",
+  "author": "mailto:john@doe.com",,
+  "timestamp": "2023-01-16T20:41:55.708329108-06:00",
+  "statements": [
+    {
+      "vulnerability": {
+        "@id": "https://nvd.nist.gov/vuln/detail/CVE-2019-17571",
+        "name": "CVE-2019-17571", 
+        "aliases": [
+            "CVE-2019-17571",
```
Comment (sudo-bmitch) [inline enhancements/opev-0015.md:37]: Is the duplication of the "name" value intentional? If so, that may need to be called out in the definition of aliases.

Comment (puerco) [reply enhancements/opev-0015.md]: I don't think the duplicate ids would be an error but it is misleading to have it twice in the example. I've removed it.

**Review by sudo-bmitch — APPROVED** [review]

File: enhancements/opev-0015.md (hunk 1 above)
Comment (puerco) [reply enhancements/opev-0015.md]: I've made it explicit in 3a059d00b75155c245b208be9394cf1a363fcdcc that the identifier in `name` can be repeated in `aliases`.

**Comment by puerco** [comment]
@lumjjb 76ef832f9122caf26cbb8475043ca7f9db606cce adds language to specify `name` as required :+1:

**Review by luhring — APPROVED** [review]

**Comment by puerco** [comment]
Lazy consensus and vote count have been reached. Thanks for your feedback, everybody!!

**Pull request merged**